### ISSUE TITLE
Can't export training program with '/' character in title

### ISSUE THREAD

**Issue by jeremiah-miller** [oliexdev/openWorkout#32]
**Describe the bug**
A training program with a '/' character in the title can be created, however when it's exported it simply fails with no explanation as to why

**To Reproduce**
1. Create a new training program
2. Rename it to anything, as long as it has a '/' character. In my case it was "r/bodyweightfitness Recommended Routine" 
3. Save new training program
4. Export training program anywhere

**Expected behavior**
I expect it to be able to export any training program I'm able to create. A couple ideas for how to do that would be blocking me from creating a training program with a '/' in the title or changing the way it stores/exports files so that file names may be slightly different from the program title

**Additional context**

**Debug log**
[openWorkout_2020-12-03_07-09.txt](https://github.com/oliexdev/openWorkout/files/5636587/openWorkout_2020-12-03_07-09.txt)

**Comment by oliexdev** [comment]
Currently this is not possible because I create a path with the workout name and on linux the char '/' is interpreted as a subfolder.

**Comment by jeremiah-miller** [comment]
Yep, I figured the best solution would be to just have an error if the string contains certain characters. Or make a function that cleans the string before using it to access the file system

**Comment by jeremiah-miller** [comment]
Yep, I figured the best solution would be to just have an error if the string contains certain characters. Or make a function that cleans the string before using it to access the file system
